analytics/app/build.gradle

```diff
@@ -29,12 +29,12 @@ repositories {
 
 dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation 'androidx.appcompat:appcompat:1.4.2'
     implementation "com.google.firebase:firebase-analytics:21.0.0"
     implementation "com.google.firebase:firebase-analytics-ktx:21.0.0"
     // Ironsource and AppLovin libraries used for ad_impression snippets
     implementation 'com.applovin:applovin-sdk:+'
-    implementation 'com.ironsource.sdk:mediationsdk:7.2.2'
+    implementation 'com.ironsource.sdk:mediationsdk:7.2.2.1'
```

Comment:
I wonder why the bot is updating ironsource. I thought it only updated certain known libraries

Reply:
Looks like it tries to update everything: https://github.com/GoogleCloudPlatform/repository-gardener/blob/master/use-latest-deps-android.sh#L35

Reply:
Oh, just remebered we have a dependency blocklist on quickstart-android that's why it doesn't update everything over there.

Thanks for looking into it, @morganchen12